### ISSUE TITLE
[React Native Gallery Preview: Modal]: Color contrast ratio for the 'This is a modal with more complex styling' text does not meet the minimum required ratio 4.5:1.

### ISSUE THREAD

**Issue by Nitin-100** [microsoft/react-native-gallery#655]
Build Details:
OS Version: 24H2 
OS Build: 26100.5074
App: React Native Gallery Preview
App version: 1.0.18.0
AT Used: Color contrast
Repro-Steps:
Launch the React Native Gallery-Preview app.
Navigate to 'Left navigation menu' button and invoke it.
Navigate to 'All samples' list item and invoke it.
Navigate to 'Modal' item and invoke it.
Navigate to 'Open modal' button under 'A modal with more complex' heading and invoke it.
Now observe the color contrast of the 'This is a modal with more complex styling' text.
Actual Result:
Color contrast ratio for the 'This is a modal with more complex styling' text does not meet the minimum required ratio 4.5:1.

Expected Results:
Color contrast ratio for the 'This is a modal with more complex styling' text should be greater than or equal to the minimum required ratio 4.5:1.
 
User Impact:
People with low vision are not able to consume the text which is having low contrast ratio.
 
MAS Reference:
[MAS 1.4.3 – Contrast (Minimum)](https://aka.ms/MAS1.4.3)
 
Trap ID:
[8.4 - Inconsistent Appearance](https://aka.ms/InconsistentAppearance)
 
“Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment section.”
